packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -13,13 +13,18 @@ export class LocalChannelStorageService implements IChannelStorageService {
     }
 
     public async read(path: string): Promise<string> {
```

Comment:
Same comment here as above - this is safe to be removed completely in favor of readBlob() and some version of readString being available.